src/main/java/com/bunq/sdk/examples/CardDebitExample.java

```diff
@@ -20,19 +22,27 @@
   private static final String EMAIL_YOUR_COMPANY = "at@at.at"; // Put your user email here
   private static final String POINTER_NAME_TEST = "test pointer";
   private static final int USER_ITEM_ID = 0; // Put your user ID here
+  private static final int MONETARY_ACCOUNT_ID = 0; // Put your monetary account ID here
   private static final int NUM_BITS_20_DECIMAL_DIGITS = 66;
   private static final int RADIX_DIGITS_AND_LATIN_LETTERS = 36;
+  private static final String CARD_PIN_ASSIGNMENT_TYPE_PRIMARY = "PRIMARY";
```

Comment:
Is there a reason why this is not an enum? Or is that impossible to generate?

Reply:
Generating ENUMs is also on the list. But its kind of impossible however. Will PM you the details.